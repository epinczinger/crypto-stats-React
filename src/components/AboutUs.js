import React from 'react';

const AboutUs = () => (
  <div>
    <h1 className="pt-20 text-center h-96 text-3xl text-red-500 font-bold">
      This site is powered
      {' '}
      <a href="https://docs.coincap.io/">by CoinCap API</a>
    </h1>
  </div>
);

export default AboutUs;
