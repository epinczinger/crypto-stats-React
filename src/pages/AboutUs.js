import React from 'react';

const AboutUs = () => (
  <div className="h60">
    <h1 className="pt-20 text-center text-3xl text-gray-700 font-bold">
      This site is powered
      {' '}
      <a href="https://docs.coincap.io/">by CoinCap API</a>
    </h1>
    <h1 className="pt-20 text-center text-2xl text-gray-700 font-bold">
      The repo is
      {' '}
      <a
        target="_blank"
        rel="noreferrer"
        href="https://github.com/epinczinger/crypto-stats-React"
        className="underline"
      >
        here.
      </a>
    </h1>
    <h1 className="pt-10 text-center text-2xl text-gray-700 font-bold">
      And the creator
      {' '}
      <a
        target="_blank"
        rel="noreferrer"
        href="https://linkedin.com/in/esteban-pinczinger/"
        className="underline"
      >
        here.
      </a>
    </h1>
  </div>
);

export default AboutUs;
