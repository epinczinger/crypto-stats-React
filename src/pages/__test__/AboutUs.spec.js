import React from "react";
import renderer from "react-test-renderer";
import AboutUs from "../AboutUs";

describe("The AboutUs component", () => {
  it("renders as expected matching snapshot", () => {
    const tree = renderer.create(<AboutUs />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
