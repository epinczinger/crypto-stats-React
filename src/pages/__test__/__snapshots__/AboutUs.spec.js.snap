// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The AboutUs component renders as expected matching snapshot 1`] = `
<div
  className="h60"
>
  <h1
    className="pt-20 text-center text-3xl text-gray-700 font-bold"
  >
    This site is powered by
     
    <a
      className="underline"
      href="https://docs.coincap.io/"
      rel="noreferrer"
      target="_blank"
    >
      CoinCap API
    </a>
  </h1>
  <h1
    className="pt-20 text-center text-2xl text-gray-700 font-bold"
  >
    The repo is
     
    <a
      className="underline"
      href="https://github.com/epinczinger/crypto-stats-React"
      rel="noreferrer"
      target="_blank"
    >
      here.
    </a>
  </h1>
  <h1
    className="pt-10 text-center text-2xl text-gray-700 font-bold"
  >
    And the creator
     
    <a
      className="underline"
      href="https://linkedin.com/in/esteban-pinczinger/"
      rel="noreferrer"
      target="_blank"
    >
      here.
    </a>
  </h1>
</div>
`;
